src/util/HeadingLevel.tsx

```diff
@@ -32,10 +32,10 @@ const HeadingLevel: React.FC<HeadingLevelProps> = ({ headingLevel, ...rest }) =>
       return <h5 {...rest} />;
     case 'h6':
       return <h6 {...rest} />;
+    default:
+      console.error(`HeadingLevel: Invalid headingLevel prop.`);
```

Comment:
I'd also print the headingLevel prop provided: `HeadingLevel: Invalid headingLevel prop: ${headingLevel}` - makes it easier for a dev to pinpoint